win32ss/user/ntuser/ntstubs.c

```diff
@@ -783,7 +783,7 @@ NtUserSetInformationThread(IN HANDLE ThreadHandle,
             Status = STATUS_SUCCESS;
             _SEH2_TRY
             {
-                ProbeForRead(ThreadInformation, sizeof(CsrPortHandle), sizeof(PVOID));
+                ProbeForRead(ThreadInformation, sizeof(CsrPortHandle), __alignof(CsrPortHandle));
```

Comment:
I was mentioned in a previous comment that `__alignof` acts on a type according to MSDN
https://learn.microsoft.com/en-us/cpp/cpp/alignof-operator?view=msvc-170
but you make it act on a variable. Is this correctly supported and does it behave correctly?